cypress/TestCases/FireboltCertification/DiscoveryLaunchHot.feature

```diff
@@ -211,4 +211,36 @@ Feature: Discovery.launch_HotLaunch
         And '3rd party app' transitions to state 'foreground'
         When 1st party app invokes the 'Firebolt' API to 'launch app with search intent int source'
-        Then 'Firebolt' platform responds to '1st party app' with 'invalid parameters for discovery launch'
\ No newline at end of file
+        Then 'Firebolt' platform responds to '1st party app' with 'invalid parameters for discovery launch'
+
+    @DiscoveryLaunch @coreSDK @sdk @transport
+    Scenario: Discovery.Launch Hot Launch - Positive Scenario: Event validation where one app is in background and one is in foreground
+        Given the environment has been set up for 'DiscoveryLaunch' tests
+        And 3rd party 'certification' app is launched with 'secondary 3rd party app' appId
+        When Test runner waits for 10 'seconds'
+        And 'secondary 3rd party app' registers for the 'discovery onNavigateTo' event using the 'Firebolt' API
+        And 'secondary 3rd party app' transitions to state 'background'
+        And 3rd party 'certification' app is launched
+        And '3rd party app' registers for the 'discovery onNavigateTo' event using the 'Firebolt' API
+        When 1st party app invokes the 'Firebolt' API to 'launch app with home intent'
+        Then 'Firebolt' platform responds to '1st party app' with 'true for discoverylaunch'
+        And '3rd party app' invokes the 'Firebolt' API to 'fetch lifecycle state'
+        Then 'Firebolt' platform responds with 'foreground for lifecycle state'
+        And 'Firebolt' platform triggers event 'onNavigateTo with home intent'
+        And 'Firebolt' platform does not trigger to 'secondary 3rd party app' event 'onNavigateTo'
+
+    @DiscoveryLaunch @coreSDK @sdk @transport
+    Scenario: Discovery.Launch Hot Launch - Positive Scenario: Event validation where one app is in inactive and one is in foreground
+        Given the environment has been set up for 'DiscoveryLaunch' tests
+        And 3rd party 'certification' app is launched with 'secondary 3rd party app' appId
+        When Test runner waits for 10 'seconds'
+        And 'secondary 3rd party app' registers for the 'discovery onNavigateTo' event using the 'Firebolt' API
+        And 'secondary 3rd party app' transitions to state 'inactive'
+        And 3rd party 'certification' app is launched
+        And '3rd party app' registers for the 'discovery onNavigateTo' event using the 'Firebolt' API
+        When 1st party app invokes the 'Firebolt' API to 'launch app with home intent'
+        Then 'Firebolt' platform responds to '1st party app' with 'true for discoverylaunch'
+        And '3rd party app' invokes the 'Firebolt' API to 'fetch lifecycle state'
+        Then 'Firebolt' platform responds with 'foreground for lifecycle state'
+        And 'Firebolt' platform triggers event 'onNavigateTo with home intent'
+        And 'Firebolt' platform does not trigger to 'secondary 3rd party app' event 'onNavigateTo'
```

Comment:
can we make these 2 test cases under one scenario outline ?

Reply:
merged in a single scenario outline